languages/de.json

```diff
@@ -50,8 +50,8 @@
   "Text Replacement" : "Text ersetzen",
   "Transformations" : "Transformationen",
   "Undo" : "Widerrufen",
-  "Use Selection for Find" : "Auswahl suchen",
-  "Use Selection for Replace" : "Auswahl für Ersetzen übernehmen",
+  "Use Selection for Find" : "In Auswahl suchen",
+  "Use Selection for Replace" : "In Auswahl ersetzen",
```

Comment:
I am missing context on this one but it sounds like when I click this, I will do a find or search not in whole document but in the selection, right?